.github/workflows/bookstore-api-optic.yml

```diff
@@ -46,3 +46,20 @@ jobs:
 
           compare_from_pr: cloud:default
           compare_from_push: cloud:default
+  verify:
```

Comment:
This would likely fold into the `run` command work we're doing - for now this can stay separate until we've consolidated the two